code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -63,6 +63,12 @@
 	if(paint_jobs)
 		paint_jobs = crate_paint_jobs
 
+	var/static/list/loc_connections = list(
+		COMSIG_ATOM_ENTERED = PROC_REF(on_entered),
+		COMSIG_ATOM_EXITED = PROC_REF(on_exited)
+	)
+	AddElement(/datum/element/connect_loc, loc_connections)
```

Comment:
Nitpick but these should be renamed to `on_loc_entered` and `on_loc_exited`, right now the procs make it seem like something entering the crate itself rather than entering the crate's turf

Reply:
Fixed